css/responsive/desktop.styl

```diff
@@ -208,14 +208,16 @@ body.single-programacion
       cursor: url('../img/dist/ui-cursor-right.png'), e-resize
       
   // DRAWER
+    
+$drawerHeaderHeight = 60.88px;
```

Comment:
Why is this set as a decimal pixel value here? Pretty sure it should be rounded rather than asking the browser to do that